roles/bats/defaults/main.yaml

```diff
@@ -17,6 +17,7 @@ bats_tests:
   - "fb-test-katello.bats"
   - "fb-katello-content.bats"
   - "fb-katello-client.bats"
+  - "fb-katello-client-global-registration.bats"
```

Comment:
I probably need to not include this for some versions of Foreman and Katello we currently support. What those would be, I am not presently sure of.

Reply:
4.0+ I believe